ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-next-postsubmits.yaml

```diff
@@ -10,9 +10,9 @@ postsubmits:
     labels:
       ci-operator.openshift.io/is-promotion: "true"
       ci-operator.openshift.io/prowgen-controlled: "true"
-      ci-operator.openshift.io/variant: "4.3"
+      ci-operator.openshift.io/variant: "4.5"
```

Comment:
Hey @markusthoemmes shouldn't the postsubmit use `4.6` since presubmit is already using `4.5`?

Reply:
This is the knative-serving repo. We don't have postsubmits there at all I believe (apart from image promotion).

Reply:
OK, sorry. I see the `4.6` references further down in the presubmit file